com.reprezen.swagedit/src/com/reprezen/swagedit/model/AbstractNode.java

```diff
@@ -0,0 +1,150 @@
+package com.reprezen.swagedit.model;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.eclipse.jface.text.BadLocationException;
+import org.eclipse.jface.text.IDocument;
+import org.eclipse.jface.text.Position;
+
+import com.fasterxml.jackson.core.JsonLocation;
+import com.fasterxml.jackson.core.JsonPointer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.base.Strings;
+import com.google.common.collect.Iterables;
+import com.reprezen.swagedit.json.JsonType;
+import com.reprezen.swagedit.json.SchemaDefinition;
+import com.reprezen.swagedit.json.SchemaDefinitionProvider;
+
+public abstract class AbstractNode {
+
+    private static SchemaDefinitionProvider provider = new SchemaDefinitionProvider();
+
+    private final JsonPointer pointer;
+    private String property;
+    private AbstractNode parent;
+    private JsonType type;
+    private JsonNode schema;
+
+    public final Set<SchemaDefinition> definitions;
```

Comment:
`definitions ` is public, but there is a corresponding `getDefinitions()`. Was the field made `public` by mistake?